greenplum-db/diskquota
# add upgrade and downgrade test

- test if works after upgrade
- test if works after downgrade
- make sure UDF/view/table is same between the new install and upgrade from old version.
- make sure the binary version is the same as SQL version. 

![image](https://user-images.githubusercontent.com/12945182/156711867-9cd4f9f3-bc4a-4a21-afd4-802e5855f594.png)
